test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1395,3 +1604,42 @@ def tearDown(self):
         # Disable the cgroups hook
         conf = {'enabled': 'False', 'freq': 30, 'event': events}
         self.server.manager(MGR_CMD_SET, HOOK, conf, self.hook_name)
+        # Cleanup the frozen jobs
+        fdir = self.paths['freezer']
+        if os.path.isdir(fdir):
+            fpath = os.path.join(fdir, 'PtlPbs')
+            if os.path.isdir(fpath):
+                jid = os.listdir(fpath)
+                if jid:
+                    for file in jid:
+                        self.logger.info("found jobdir %s" % file)
+                        jpath = os.path.join(fpath, file)
+                        freezer_file = os.path.join(jpath, 'freezer.state')
```

Comment:
Combine the above two join() calls

Reply:
I am using jpath further down to remove those jobdir.